router/server.go

```diff
@@ -465,6 +478,27 @@ func (s *Server) runHttpBridgeServers() error {
 	return firstErr
 }
 
+func (s *Server) addPortToConfigService() error {
```

Comment:
Yes, I think it's worthwhile adding such methods to serviceconfigsgo. I regret that the API wasn't built to be
`/config-service/variable/name/value`, because it would have made the API calls simpler.